charts/ssi-credential-issuer/values.yaml

```diff
@@ -86,11 +86,14 @@ migrations:
       memory: 200M
   seeding:
     testDataEnvironments: ""
-    testDataPaths: "Seeder/Data"
+    testData:
+      useDefaultTestData: false
+      # -- ConfigMap containing json files for the tables to seed, e.g. use_cases.json, verified_credential_external_type_detail_versions.test.json, etc.
+      configMap: ""
+      # -- Filename identifying the test data files e.g. for companies.test.json the value would be "test"
+      filename: ""
```

Comment:
```suggestion
    testData:
      useDefault: false
      useOwnConfigMap:
        configMap: ""
        filename: ""
```

Reply:
adjusted

Reply:
unfortunately I can't confirm, I don't see `useOwnConfigMap`

Reply:
missed that... adjusted